docs/src/dev-guide/components-core/index.md

```diff
@@ -25,29 +25,31 @@ tools/scripts/deps-download/init.sh
 
 ### Source Dependencies
 
-We use both git submodules and third-party source packages. To download all, run this `task`
-command:
+To download and build all dependencies required by the core, run this `task` command:
 
 ```shell
 task deps:core
 ```
 
-This will download:
-* [abseil-cpp](https://github.com/abseil/abseil-cpp) (20230802.1)
-* [ANTLR](https://www.antlr.org) (v4.13.1)
-* [Catch2](https://github.com/catchorg/Catch2.git) (v2.13.7)
-* [date](https://github.com/HowardHinnant/date.git) (v3.0.1)
-* [fmt](https://github.com/fmtlib/fmt) (v8.0.1)
-* [json](https://github.com/nlohmann/json.git) (v3.11.3)
-* [log-surgeon](https://github.com/y-scope/log-surgeon) (f801a3f)
-* [outcome](https://github.com/ned14/outcome) (v2.2.9)
-* [simdjson](https://github.com/simdjson/simdjson) (v3.6.3)
-* [spdlog](https://github.com/gabime/spdlog) (v1.9.2)
-* [SQLite3](https://www.sqlite.org/download.html) (v3.36.0)
-* [uftcpp](https://github.com/nemtrif/utfcpp.git) (v4.0.6)
-* [yaml-cpp](https://github.com/jbeder/yaml-cpp.git) (v0.7.0)
-* [yscope-log-viewer](https://github.com/y-scope/yscope-log-viewer.git) (969ff35)
-* [ystdlib-cpp](https://github.com/y-scope/ystdlib-cpp.git) (2ac1757)
+The command above will fetch, build, and install the following exact versions:
+
+| Library            | Version/commit | Notes                               |
+|--------------------|----------------|-------------------------------------|
+| absl               | 20230802.1     | renamed from abseil-cpp             |
+| antlr4             | v4.13.1        | JAR only                            |
+| catch2             | v2.13.7        |                                     |
+| date               | v3.0.1         |                                     |
+| fmt                | v8.0.1         |                                     |
+| nlohmann_json      | v3.11.3        | renamed from json                   |
+| log-surgeon        | f801a3f        |                                     |
+| outcome            | v2.2.9         |                                     |
+| simdjson           | v3.6.3         |                                     |
+| spdlog             | v1.9.2         |                                     |
+| sqlite3            | v3.36.0        |                                     |
+| utfcpp             | v4.0.6         |                                     |
+| yaml-cpp           | v0.7.0         | target name changed to yaml-cpp     |
+| yscope-log-viewer  | 969ff35        |                                     |
+| ystdlib            | 2ac1757        | renamed from ystdlib-cpp            |
```

Comment:
Good idea to use a table. Let's:

* keep the original package name, since the internal task name shouldn't really matter to the developer (they can't invoke it without doing something special, anyway). So in that case, I don't think we'd need the "Notes" column.
* keep the links to the each package. If you want to refactor things so that they read nicely in Markdown, you can use link references like we do for `[Task]`.